lib/scrabble.rb

```diff
@@ -9,5 +10,63 @@ def point_values
       "U"=>1, "V"=>4, "W"=>4, "X"=>8,
       "Y"=>4, "Z"=>10
     }
+    @multi_score
+    @multiplier_array
+    @seven_bonus = 0
   end
+  def score(letter)
+    if letter == nil
```

Comment:
You could combine the first two branches into something like this: `if letter == nil || letter == ""`